src/phasorpy/phasor.py

```diff
@@ -2998,6 +3000,168 @@ def phasor_filter_median(
     return mean, real, imag
 
 
+def phasor_filter_wavelet(
+    mean: ArrayLike,
+    real: ArrayLike,
+    imag: ArrayLike,
+    /,
+    *,
+    sigma: float = 2.0,
+    levels: int = 1,
+    harmonic: int | Sequence[int] | Literal['all'] | str | None = None,
+    skip_axis: int | Sequence[int] | None = None,
+    normalize: bool = False,
+    samples: int = 1,
+    nan_safe: bool = True,
+) -> tuple[NDArray[Any], NDArray[Any], NDArray[Any]]:
+    """Return wavelet-filtered phasor coordinates.
+
+    Parameters
+    ----------
+    mean : array_like
+        Intensity of phasor coordinates.
+    real : array_like
+        Real component of phasor coordinates to be filtered.
+    imag : array_like
+        Imaginary component of phasor coordinates to be filtered.
+    sigma : float, optional
+        Significance level to test the difference between two phasors, given
+        in terms of the equivalent 1D standard deviations. sigma=2 corresponds
+        to ~95% (or 5%) significance.
+    levels : int, optional
+        Number of levels for the wavelet decomposition.
+        Controls the maximum averaging area, which has a length of 2**level.
+    harmonic : int, sequence of int, or 'all', default: 1
+        Harmonics included in `real` and `imag`.
+        If an integer, the harmonics at which `real` and `imag` were acquired
+        or calculated.
+        If a sequence, the harmonics included in the first axis of `real` and
+        `imag`.
+        If `'all'`, the first axis of `real` and `imag` contains lower
+        harmonics.
+        The default is the first harmonic (fundamental frequency).
+    skip_axis : int or sequence of int, optional
+        Axes in `mean` to exclude from filter.
+        By default, all axes except harmonics are included.
+    normalize : bool, optional
+        By default (False), expects and returns normalized phasor coordinates.
+        If True, unnormalized phasor coordinates are expected as input.
```

Comment:
OK for now (while still developing), but as discussed, I do not think phasorpy functions should take/return un-normalized phasor coordinates.